.github/copilot-instructions.md

```
@@ -0,0 +1,101 @@
+# OneCAD AI Agent Instructions
+
+## Project Identity
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)